Sprint-1/errors/4.js

```diff
@@ -1,2 +1,4 @@
-const 12HourClockTime = "20:53";
-const 24hourClockTime = "08:53";
\ No newline at end of file
+const HourClockTime12 = "20:53";
```

Comment:
Have you also noticed the variable names do not quite match the values assigned to the variable?

Reply:
Oh, you're right. thank you